Makefile

```diff
@@ -147,6 +153,17 @@ DOCKERSAVE=$(DOCKERCMD) save
 DOCKERCOMPOSECMD=$(shell which docker-compose)
 DOCKERTAG=$(DOCKERCMD) tag
 
+# according to the build platform, replace the relevant parameters.
```

Comment:
If compiling the image of the arm architecture, replace the build image parameters `arm` suffix